src/freenas/etc/ix.rc.d/ix-ups

```diff
@@ -54,7 +54,9 @@ generate_upsmon()
 	while eval read $f; do
 
 		user=`echo ${ups_monuser}|sed -E 's/([#$])/\\\1/g'`
-		passwd=`/usr/local/bin/midclt call pwenc.decrypt ${ups_monpwd}|sed -E 's/([#$])/\\\1/g'`
+		if [ -n "${ups_monpwd}" ]; then
```

Comment:
Should we have an else here as `passwd` won't be declared if this if evaluates to false ?

Reply:
shell doesnt care for that, it will output empty passwd